### ISSUE TITLE
Slowness effect removed after some seconds

### ISSUE THREAD

**Issue by thebigsmileXD** [iTXTech/Genisys#1849]
@dktapps here is the issue.

The slowness effect does only work once after server started, shows up just fine client side.

**Comment by thebigsmileXD** [comment]
There we go, found the reason. When you start sprinting, slowness gets totally ignored. Even when a new effect is applied, no change

**Comment by Sandertv** [comment]
Yeah I can reproduce this. When adding a speed effect after a slowness effect the speed effect does not make a change either.
